XmlFormatter/src/Windows/HotfolderEditor.cs

```diff
@@ -101,14 +104,19 @@ private void FillSettingWindow()
             CB_Mode.SelectedIndex = 0;
             if (editMode)
             {
+                bool foundEntry = false;
                 for (int i = 0; i < CB_Formatter.Items.Count; i++)
                 {
-                    string item = CB_Formatter.Items[i].ToString();
-                    if (item == hotfolder.FormatterToUse.GetType().Name)
+                    if (CB_Formatter.Items[i] is ComboboxPluginItem item)
                     {
-                        CB_Formatter.SelectedIndex = i;
-                        break;
+                        if (item.Type.FullName == hotfolder.FormatterToUse.GetType().FullName)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Merge this if statement with the enclosing one.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5494671)